eq-author/cypress/integration/authenticated/piping_spec.js

```diff
@@ -15,6 +15,7 @@ const METADATA = "example_metadata";
 const clickPipingButton = () =>
   cy
     .get(testId("piping-button"))
+    .should("have.length", 1)
```

Comment:
Are these cypress changes use to slow it down so it can wait for something or were they added for debugging?

Reply:
Sometimes when focusing on one RTE to another (eg. Question Title to Question Description).  The animation which fades the blurred RTE toolbar is a bit slow, therefore you end up with 2 x interactable piping buttons.  This test ensures we only interact with the piping button when there is only 1 x piping button.

Reply:
I'm not too sure why this PR highlighted this issue. 😕